### DISCUSSION TITLE
Is this project active?

### DISCUSSION THREAD

**Discussion by ghost** [nagadomi/waifu2x#434] [Q&A] [answered]
Hi everyone.

I want to help, but I want to know if this project is active. *Is this project active?*

**Comment by 1374363910** [comment]
这是章泽鑫的邮箱，我收到你的邮件了。

**Comment by nagadomi** [comment, marked as answer]
the project is active in another repo.
but do not want help from people who post this kind of question.

**Reply by ghost** [reply]
Hi nagadomi.

> the project is active in another repo.
> but do not want help from people who post this kind of question.

So, sorry. I'm new to git and I don't know how to use github, I want to help and contribute.

**Reply by nagadomi** [reply]
If there are bug fixes or changes, just send a Pull Request.
If the changes are objectively correct, it will be merged, otherwise it will depend on my feeling.

**Reply by ghost** [reply]
Hi nagadomi.

> If there are bug fixes or changes, just send a Pull Request.
> If the changes are objectively correct, it will be merged, otherwise it will depend on my feeling.

thank you for feedback again. I am happy with your comment and answer. This is very important to me.
